README.rst

```diff
@@ -39,7 +39,7 @@ to see starfish in action on particular assay types. Or, browse our our notebook
  MERFISH              [x]         [x] mer_             in process
  ISS                  [x]         [x] iss_             in process
  osmFISH              [x]         [x] osm_             [ ]
- smFISH               [x]         [x]                  [ ]
+ smFISH               [x]         [x] 3ds_             [ ]
```

Comment:
place an x under the last column? because we have an entire experiment processed for smFISH?